packaging/prerelease/build_app.sh

```diff
@@ -22,7 +23,7 @@ if [ "$gopath" = "" ]; then
 fi
 
 if [ "$platform" = "" ]; then
-  echo "No PLATFORM. Yo can specify darwin, linux or windows."
```

Comment:
Do you think it would make sense to autodetect platform? We don't support cross-platform builds, do we?

Though now that I think about it, this script is only ever used for OSX, so probably "darwin" should just be hardcoded?